packages/conf-libgl/conf-libgl.1/opam

```diff
@@ -0,0 +1,19 @@
+opam-version: "2.0"
+maintainer: "Marek Kubica <marek@xivilization.net>"
+authors: ["ARB"]
+homepage: "https://www.khronos.org/opengl"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+build: [
+  ["pkg-config" "--exists" "gl"]
+]
+depends: [
+  "conf-pkg-config" {build}
+]
+depexts: [
+  ["mesa-common-dev"] {os-family = "debian"}
+  ["mesa-libGL-devel"] {os-distribution = "fedora"}
+  ["mesa-dev"] {os-distribution = "alpine"}
+  ["libgl"] {os-family = "arch"}
```

Comment:
```
+   ["libglvnd" {os = "freebsd"}
```
according to https://www.freshports.org/graphics/libglvnd
(prev. libgl was apparently part of https://www.freshports.org/graphics/mesa-libs
and before that https://www.freshports.org/graphics/libGL)

Reply:
Yeah, this one is a bit weird, in Arch it is also libglvnd which seems mostly like a library that redirects to your actual GL implementation, be it Mesa, NVIDIA or something else. I'll try and see what the CI says!

Reply:
CI is happy with that; resolving :)